modules/apps/data-engine/source-formatter-suppressions.xml

```diff
@@ -2,6 +2,7 @@
 
 <suppressions>
 	<source-check>
+		<suppress checks="JavaVariableTypeCheck" files="data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/util/content/type/TestDataDefinitionContentType\.java" />
```

Comment:
@RenanUmV @rafaprax We can put the field in a util so that we can avoid adding an SF suppression.

By the way, this component is from a test util, which will not affect portal startup. Do we really need to change it?